cli_tools/common/distro/distro.go

```diff
@@ -53,6 +53,15 @@ var osFlagExpression = regexp.MustCompile(
 		// License is optional. The only value is `byol`.
 		"(?:-(?P<license>byol))?$")
 
+var architectureLookup = map[string]string{
```

Comment:
What is this lookup table mapping from? Consider a better variable name or a comment to provide more context

Reply:
good point -- consolidated with the map below and renamed to  "standardArchitectures"